roles/openshift_logging_mux/templates/mux.j2

```diff
@@ -46,6 +46,13 @@ spec:
 {% if mux_memory_limit is not none %}
             memory: "{{mux_memory_limit}}"
 {% endif %}
+          requests:
+{% if mux_cpu_request is not none %}
+            cpu: "{{mux_cpu_request}}"
+{% endif %}
+{% if mux_memory_request is not none %}
+            memory: "{{mux_memory_request}}"
+{% endif %}
```

Comment:
Is it possible that the `requests` section can have no values?  Is the `not none` test sufficient to exclude the empty string value `''`?